examples/fabric-bridge-app/fabric-bridge-common/fabric-bridge-app.matter

```diff
@@ -1647,6 +1698,21 @@ endpoint 0 {
     handle command KeySetReadAllIndices;
     handle command KeySetReadAllIndicesResponse;
   }
+
+  server cluster CommissionerControl {
```

Comment:
@tehampson Is that OK, that the CommissionerControl is on endpoint 0? In the spec, the CommissionerControl should be on Aggregator endpoint, which is endpoint 1 here.